spec/graphql/relay/array_connection_spec.rb

```diff
@@ -187,14 +187,74 @@ def get_page_info(result)
         assert_equal(["Yavin", "Echo Base"], get_names(result))
         assert_equal(false, get_page_info(result)["hasPreviousPage"], "hasPreviousPage is false when last is not specified")
 
-        third_cursor = "Mw=="
+        fourth_cursor = "Mw=="
```

Comment:
On the table you posted in the description, `"Mw=="` is the third row. Are you sure we should rename this?